tests/test/test_rks_administration_api.py

```diff
@@ -130,6 +128,20 @@ def test_create_group(self, admin_api):
         # reset admin_api with root token and delete group
         admin_api.api_client.configuration.api_key["X-Vault-Token"] = utils.ADMIN_TOKEN
 
+        response, status, headers = admin_api.delete_group_with_http_info("fakecdn1")
+        assert status == 204, "Delete group does not return 204 as expected"
+
+    @pytest.mark.selected
```

Comment:
Instead of using marks to run specific tests, which stays in the code and are only used locally when debugging, we should use pytest `-k` flag -> [doc](https://docs.pytest.org/en/latest/example/markers.html#using-k-expr-to-select-tests-based-on-their-name)

Example use:
`pytest -k "test_update_group_token"` Run a single test
`pytest -k "group_token"` Run all tests with group_token in their name (currently test_get_group_token, test_update_group_token, test_renew_group_token and test_delete_group_token)

This will also remove the following warning when running tests:
```
/tests/test/test_rks_administration_api.py:134: PytestUnknownMarkWarning: Unknown pytest.mark.selected - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
```